lib/apparmortest.pm

```diff
@@ -169,7 +170,7 @@ sub aa_status_stdout_check {
     my $total_line = script_output("aa-status | grep 'profiles are in' | grep $profile_mode | cut -d ' ' -f1");
     my $lines = $start_line + $total_line;
 
-    assert_script_run("aa-status | head -$lines | tail -$total_line | sed 's/[ \t]*//g' | grep -x $profile_name");
+    script_output("aa-status | head -$lines | tail -$total_line | sed 's/[ \t]*//g' | grep -x $profile_name");
```

Comment:
No!

Reply:
Because of control characters characters

Reply:
> Did you read the commit message ? This will get back to broken state as it was before!

Ahhh I get it now, can you add a comment above the line instead? same below